views/index.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Welcome to Chitter! </h1>
+<p> See your timeline <a href="http://localhost:9292/timeline">here</a>
+<p> Complete the form below to sign up and create a free account <p>
+
+<form action='/' method="POST">
+  Full name: <input type="text" name="name" placeholder="Sam Morgan" /><br>
+<br>
+  Username: <input type="text" name="username" placeholder="username" /><br>
+<br>
+  Email address: <input type="text" name="emailaddress" placeholder="emailaddress@mail.com" /> <br>
+<br>
+  Password: <input type="text" name="password" placeholder="password" /> <br>
```

Comment:
one thing i discovered was you can call the type="password" in the input and it changes the user input to * so you can't see whats being typed. Thought that was cool. You can also specify how long or short the password needs to be.

Same goes for type="email" which makes it require an @ sign.